src/transforms/sample.rs

```diff
@@ -14,6 +14,8 @@ use crate::{
     transforms::{FunctionTransform, OutputBuffer, Transform},
 };
 
+pub static SAMPLE_RATE_FIELD: &str = "sample_rate";
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Will want to use `const` over `static` here since the variable doesn't need to be modified.

```suggestion
pub const SAMPLE_RATE_FIELD: &str = "sample_rate";
```